content/concepts/multiplex/overview.md

```diff
@@ -9,24 +9,36 @@ aliases:
 
 ## Overview
 
-Stream Multiplexing (_stream muxing_) is a way of sending multiple streams of data over one
-communication link. It combines multiple signals into one unified signal so it can be transported
-'over the wires', then it is demulitiplexed (_demuxed_) so it can be output and used by separate
-applications. This is done to share a single TCP connection using unique port numbers to distinguish
-streams, between the multiple proceeses (such as kademlia and gossipsub) used by applications (such as IPFS)
-to make connection and transmission more efficient.
-
-With muxing, libp2p applications may have many separate streams of communication between peers, as well as
-have multiple concurrent streams open at the same time with a peer. Stream multiplexing allows us to initialize
-and use the same [transport](../../transports/overview) connection across the lifetime of our interaction with a
-peer. With muxing, we also only need to deal with [NAT traversal](../../nat/overview) once to be able to open as
-many streams as we need, since they will all share the same underlying transport connection. Applications can enable
-support for multiple multiplexers, which will allow you to fall back to a widely-supported multiplexer if a preferred
-choice is not supported by a remote peer.
+Stream multiplexing (or stream muxing) is a method to send multiple streams of
+data over a single communication link. It allows peers to use the same connection
+across the lifetime of their interaction with another peer.
+
+In particular, stream muxing is done to share a single connection between multiple
+protocols used by applications to make connection and transmission more efficient.
+This is especially useful when doing NAT traversal or hole punching,
+as it allows for efficient use of limited resources. This is also used outside of
+libp2p, for example, HTTP/2 introduced streams into HTTP, allowing for many HTTP
+requests in parallel on the same connection.
+
+## Muxers in libp2p
+
+Establishing a libp2p connection can be expensive and take a few round-trips.
+Peers therefore aim to establish a connection once, then run many streams over
+the same connection. Opening has a low resource overhead and does not impose any
+latency penalty.
```

Comment:
This is kind of repeating what we had in the section before. I suggest to move the sentence about stream opening to that section.